app/src/main/java/com/randomappsinc/studentpicker/home/DeleteListDialog.java

```diff
@@ -0,0 +1,46 @@
+package com.randomappsinc.studentpicker.home;
+
+import android.app.Dialog;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.FragmentManager;
+
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.randomappsinc.studentpicker.R;
+
+public class DeleteListDialog extends DialogFragment {
+
+    public interface Listener {
+        void onDeleteListConfirmed(int position);
+    }
+
+    private FragmentManager fragmentManager;
+    private Listener listener;
+    private int position;
+
+    public DeleteListDialog(FragmentManager fragmentManager, Listener listener) {
```

Comment:
Because I am extending the DialogFragment I need to pass the FragmentManager in the show method to use the context in onCreateDialog, if I make an instance of the material dialog inside the class it will make a NullPointerException.